backend/app/treasury/storage_helpers/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .storage_helpers import retrieve_treasuries_metadata
+from .treasury_cacher import get_and_store_treasury_list
```

Comment:
(For some reason I'm not able to comment on the `treasury_cacher.py` file so here is my remark on this file)

I have the same remark as in backend/app/token_whitelists/storage_helpers/storage_helpers.py  
The treasury_cacher.py module is really an adapter to cryptostats, so it should be named accordingly.

Another important remark is that this adapter should take care only about cryptostats, not anything else. So we should not allow it to make calls to the `store_treasuries_metadata` function which stems from another adapter. As a rule of thumb, whatever function is trying to "merge" several adapter together should be in the core of the application. More concretely, `get_and_store_treasury_list` should be in your core module, which is `whitelists.py`.

Reply:
I'm realizing that we're not respecting this rule in the rest of the codebase, as we're accessing the "global" `db` object. That basically means that our other adapters are already too knowledgeable. I'm not sure yet how to address the problem of the global `db` object.

Reply:
> Do you see this as issue due to the fact that we are importing the db into the modules like from .... import db or that we're using a "global db" in the first place?

My concern is with the code: `import db`, not really with the existence of a global object.

We can totally have a singleton db object if that helps.